projects/plugins/jetpack/sal/class.json-api-post-base.php

```diff
@@ -209,41 +391,68 @@ public function get_other_urls() {
 		return (object) $other_urls;
 	}
 
+	/**
+	 * Calls the WPCOM_JSON_API_Links get_site_link() function to generate a site link endpoint URL.
+	 *
+	 * @return string Endpoint URL including site information.
+	 */
 	protected function get_site_link() {
 		return $this->links->get_site_link( $this->site->get_id() );
 	}
 
+	/**
+	 * Calls the WPCOM_JSON_API_Links get_post_link() function to generate a posts endpoint URL.
+	 *
+	 * @param string $path  Optional path to be appended to the URL.
+	 * @return string Endpoint URL including post information.
+	 */
 	protected function get_post_link( $path = null ) {
 		return $this->links->get_post_link( $this->site->get_id(), $this->post->ID, $path );
 	}
 
+	/**
+	 * Returns an array of user and post specific social media post URLs.
+	 *
+	 * @return array
+	 */
 	public function get_publicize_urls() {
-		$publicize_URLs = array();
+		$publicize_urls = array();
 		$publicize      = get_post_meta( $this->post->ID, 'publicize_results', true );
 		if ( $publicize ) {
 			foreach ( $publicize as $service => $data ) {
 				switch ( $service ) {
-				case 'twitter' :
-					foreach ( $data as $datum ) {
-						$publicize_URLs[] = esc_url_raw( "https://twitter.com/{$datum['user_id']}/status/{$datum['post_id']}" );
-					}
-					break;
-				case 'fb' :
-					foreach ( $data as $datum ) {
-						$publicize_URLs[] = esc_url_raw( "https://www.facebook.com/permalink.php?story_fbid={$datum['post_id']}&id={$datum['user_id']}" );
-					}
-					break;
+					case 'twitter':
+						foreach ( $data as $datum ) {
+							$publicize_urls[] = esc_url_raw( "https://twitter.com/{$datum['user_id']}/status/{$datum['post_id']}" );
+						}
+						break;
+					case 'fb':
+						foreach ( $data as $datum ) {
+							$publicize_urls[] = esc_url_raw( "https://www.facebook.com/permalink.php?story_fbid={$datum['post_id']}&id={$datum['user_id']}" );
+						}
+						break;
 				}
 			}
 		}
-		return (array) $publicize_URLs;
+		return (array) $publicize_urls;
 	}
 
+	/**
+	 * Returns a string with the page's custom template metadata.
+	 *
+	 * @return string
+	 */
 	public function get_page_template() {
 		return (string) get_post_meta( $this->post->ID, '_wp_page_template', true );
 	}
 
-	// note this is overridden in jetpack-shadow
+	/**
+	 * Returns a string representing the source URL of a post's featured image (or an empty string otherwise).
+	 *
+	 * Note - this is overridden in jetpack-shadow
```

Comment:
So it turns out "jetpack-shadow" is a wpcom thing. Not sure it's worth mentioning in the doc comment.